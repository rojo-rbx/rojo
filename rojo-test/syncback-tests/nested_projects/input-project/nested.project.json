{
  "name": "Nested",
  "tree": {
    "$className": "Configuration",
    "StringValue": {
      "$path": "string_value.txt"
    },
    "BoolValue": {
      "$className": "BoolValue"
    }
  }
}