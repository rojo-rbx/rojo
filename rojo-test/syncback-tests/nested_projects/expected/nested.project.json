{
  "name": "Nested",
  "tree": {
    "$className": "Configuration",
    "BoolValue": {
      "$className": "BoolValue",
      "$properties": {
        "Value": true
      }
    },
    "StringValue": {
      "$path": "string_value.txt"
    }
  }
}